pkg/server/server_test.go

```diff
@@ -653,9 +729,9 @@ var _ = Describe("ApplicationsServer", func() {
 				}
 
 				appFactory := &apputilsfakes.FakeAppFactory{}
-				appFactory.GetAppServiceStub = func(ctx context.Context, name, namespace string) (app.AppService, error) {
-					return app.New(ctx, nil, nil, nil, nil, nil, kubeClient, nil), nil
-				}
+				// appFactory.GetAppServiceStub = func(ctx context.Context, name, namespace string) (app.AppService, error) {
```

Comment:
clean this up

Reply:
> clean this up

Is there a nicer way to say this?

Reply:
sorry, didn't mean to be rude I wrote it more like an #TODO comment.